pygeos/linear.py

```diff
@@ -140,3 +146,31 @@ def shared_paths(a, b, **kwargs):
     <pygeos.Geometry GEOMETRYCOLLECTION (MULTILINESTRING EMPTY, MULTILINESTRING ...>
     """
     return lib.shared_paths(a, b, **kwargs)
+
+
+def shortest_line(a, b, **kwargs):
+    """
+    Returns the shortest line between geom1 and geom2.
+
+    The resulting line consists of two points, representing the nearest
+    points between the geometry pair. The line always starts in geom1 and
+    ends in geom2. The endpoints of the line will not necessarily be existing
+    vertices of geom1 and geom2, but can be any point along a line segment
+    (if the input geometries are lines or polygons).
+
+    Parameters
+    ----------
+    a : Geometry or array_like
+    b : Geometry or array_like
+    **kwargs
+        For other keyword-only arguments, see the
+        `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
+
+    Examples
```

Comment:
Can you add a "See also" section before the Examples that refers to `prepare`? The predicate functions have that, I think it is good to mention that preparing geometries will help here (under the condition that GEOS>=3.9).